README.rst

```diff
@@ -256,66 +259,66 @@ For completeness, its usage:
+                         [-u USERNAME] [-p PASSWORD] [-c COMMENT]
+                         [--config-file CONFIG_FILE] [--cert path]
+                         [--client-cert path]
+                         package
+
+   positional arguments:
+     package               File from which we read the package metadata.
+
+   optional arguments:
+     -h, --help            show this help message and exit
+     -r REPOSITORY, --repository REPOSITORY
+                           The repository (package index) to register the package
+                           to. Should be a section in the config file. (Can also
+                           be set via TWINE_REPOSITORY environment variable.)
+                           Initial package registration no longer necessary on
+                           pypi.org:
+                           https://packaging.python.org/guides/migrating-to-pypi-
+                           org/
+     --repository-url REPOSITORY_URL
+                           The repository (package index) URL to register the
+                           package to. This overrides --repository. (Can also be
+                           set via TWINE_REPOSITORY_URL environment variable.)
+     -u USERNAME, --username USERNAME
+                           The username to authenticate to the repository
+                           (package index) as. (Can also be set via
+                           TWINE_USERNAME environment variable.)
+     -p PASSWORD, --password PASSWORD
+                           The password to authenticate to the repository
+                           (package index) with. (Can also be set via
+                           TWINE_PASSWORD environment variable.)
+     --non-interactive     Do not interactively prompt for username/password
+                           if the required credentials are missing. (Can also
+                           be set via TWINE_NON_INTERACTIVE environment
+                           variable.)
+     -c COMMENT, --comment COMMENT
+                           The comment to include with the distribution file.
+     --config-file CONFIG_FILE
+                           The .pypirc config file to use.
+     --cert path           Path to alternate CA bundle (can also be set via
+                           TWINE_CERT environment variable).
+     --client-cert path    Path to SSL client certificate, a single file
+                           containing the private key and the certificate in PEM
+                           format.
 
 Environment Variables
 ^^^^^^^^^^^^^^^^^^^^^
 
 Twine also supports configuration via environment variables. Options passed on
 the command line will take precedence over options set via environment
 variables. Definition via environment variable is helpful in environments where
-it is not convenient to create a `.pypirc` file, such as a CI/build server, for
-example.
+it is not convenient to create a ``.pypirc`` file, such as a CI/build server,
+for example,
```

Comment:
Thank you for the catch!  BTW to avoid heated discussion over a pretty much non-issue, I'll revert the whitespace changes and rebase the branch.  Thank you both for putting up with me on this PR.